File: taller-1/python/taller1-notas.py
nombre = input("Ingrese el nombre del estudiante: ")

notas = []
suma = 0

for i in range(1,4):
    while True:
        try:
            nota = float(input(f"Ingrese la nota {i}: "))
            if 0<= nota <=5:
                break
            else:
                print("Error la nota deb estar entre 0 y 5")
        except ValueError:
            print("Error: Ingrese un numero valido")

    notas.append(nota)
    suma += nota

promedio = suma / 3
estado = "APROBADO" if promedio >= 3.0 else "REPROBADO"

print (f"{nombre} tiene un promedio de {promedio:.2f} y esta {estado}")

File: taller-1/js/taller1-notas.js
const prompt = require('prompt-sync')();

let nombre = prompt('Ingrese el nombre del estudiante: ')
let notas = []
let suma=0;

for (let i =1; i<=3; i++){
    let nota;

    do{
         nota = parseFloat(prompt(`Ingerese la nota ${i}: `))
        if(nota <0 || nota > 5 || isNaN(nota)){
            console.log('Error: la nota de estar entre 0 y 5');
        }
}while(nota <0 || nota > 5 || isNaN(nota));

    notas.push(nota)
    suma += nota;

}

let promedio = suma/3;
let estado = promedio >= 3.0 ? "APROBADO":"REPROBADO"

console.log(`${nombre} tiene un promedio de ${promedio.toFixed(2)} y esta ${estado}`)
